submitted_models/marble_husky_sensor_config_3/specifications.md

```diff
@@ -0,0 +1,86 @@
+<!---This is a Markdown description of a robot model submitted for inclusion in the DARPA Subterranean Challenge Technology Repository -->
+
+# MARBLE Husky Sensor Config 3
+This specifications.md file is a description and proof of virtual model validation for the MARBLE Husky with Sensor Configuration 1. This robot may be launched using an ign launch command with the variable name `marble_husky_sensor_config_3`.
+
+## Description
+This configuration is based on Clearpath Robotics Husky ground robot. The marble sensor suite is located at the front of the husky and includes stationary sensors and some sensors and a light on a pan/tilt mechanism.  
+
+## Usage Instructions
+The robot can be used in the same manner as the COSTAR Husky robot.  It accepts twist messages on the cmd_vel topic.  The gimbal can be controlled using the test_gimbal.sh script (in this folder).  There are also scripts to print the current angles from the pan and tilt axes of the gimbal (pan_echo.sh and tilt_echo.sh).
+
+The gimbal can be controlled via the following topics:
+pan_tilt/pan_rate_cmd_double (send a std_msgs/Double message)
+pan_tilt/tilt_rate_cmd_double (send a std_msgs/Double message)
+
+The position of the pan and tilt axes can be accessed through the joint_state topic.  The name of each axis is available inside that message, but as of today the pan and tilt can be accessed as follows:
+Pan: joint_state/position[4]
+Tilt: joint_state/position[5]
+
+## Usage Rights
+The same Rights are granted for the configuration as for the COSTAR Husky. No additional restrictions have to be taken into account for this configuration.
+
+### Cost and Scale
+The MARBLE Husky has the following estimate commercial cost components:
+* Base vehicle: $25,000
+* Sensor suite: $12,000
+* Compute/Support Electronics: $4,000
+* Total: ~ $41,000
+
+Its weight is approximately 85 lbs (two-person heft).
+
+### Sensors
+This MARBLE Husky with sensor configuration 1 includes the following sensors. The specifications for these instruments are provided below in the [Validation Links](#validation_links) section.
+
+The following specific sensors are declared payloads of this vehicle:
+
+* Trossen ScorpionX MX-64 Robot Turret, modeled by JointStateController and JointStatePublisher plugins.
+* D435i RGBD Camera (x3), modeled by rgbd_camera plugin
```

Comment:
There is only one RGBD camera int he sdf file, is this correct?

Reply:
This is correct. The specifications.md file has been updated to more accurately reflect the vehicle config.